src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -1004,7 +1010,17 @@ void *os::Bsd::dlopen_helper(const char *filename, int mode) {
     // flags. Silently fix things now.
     int rtn = fesetenv(&default_fenv);
     assert(rtn == 0, "fesetenv must succeed");
-    assert(IEEE_subnormal_handling_OK(), "fsetenv didn't work");
+    bool ieee_handling_after_issue = IEEE_subnormal_handling_OK();
```

Comment:
This is a misleading name. It should be something explicit like `ieee_handling_succeeded`. Still, I suppose it's too late now.

Reply:
okay why not ieee_handling_succeeded; I can change the variable name in the follow  up  JDK-8321017 .